diff --git a/node_modules/canvas-datagrid/lib/component.js b/node_modules/canvas-datagrid/lib/component.js
index 40fea42..5235e21 100644
--- a/node_modules/canvas-datagrid/lib/component.js
+++ b/node_modules/canvas-datagrid/lib/component.js
@@ -235,17 +235,6 @@ export default function () {
       attributes: true,
       subtree: true,
     });
-    Array.prototype.forEach.call(
-      document.querySelectorAll('style'),
-      function (el) {
-        observer.observe(el, {
-          characterData: true,
-          childList: true,
-          attributes: true,
-          subtree: true,
-        });
-      },
-    );
   };
   return component;
 }
diff --git a/node_modules/canvas-datagrid/lib/main.js b/node_modules/canvas-datagrid/lib/main.js
index 0a74d76..42a675f 100644
--- a/node_modules/canvas-datagrid/lib/main.js
+++ b/node_modules/canvas-datagrid/lib/main.js
@@ -61,6 +61,7 @@ function Grid(args) {
     self.parentNode = self.shadowRoot;
   }
   self.init();
+  self.intf.self = self;
   return self.intf;
 }
 if (window.HTMLElement) {
diff --git a/node_modules/canvas-datagrid/package.json b/node_modules/canvas-datagrid/package.json
index 5338514..a45937f 100644
--- a/node_modules/canvas-datagrid/package.json
+++ b/node_modules/canvas-datagrid/package.json
@@ -2,8 +2,7 @@
   "name": "canvas-datagrid",
   "version": "0.3.1",
   "description": "Canvas based data grid web component. Capable of displaying millions of contiguous hierarchical rows and columns without paging or loading, on a single canvas element.",
-  "main": "./dist/canvas-datagrid.js",
-  "module": "./dist/canvas-datagrid.module.js",
+  "main": "./lib/main.js",
   "scripts": {
     "build": "rollup -c && webpack",
     "build:docs": "jsdoc -c ./jsdoc.conf.js && cp dist/canvas-datagrid.* docs/ && cp -r tutorials/* images docs/",
